fastai/fastaiv1_simple.py

```diff
@@ -70,11 +70,11 @@ def objective(trial):
         model,
         silent=True,
         metrics=[vision.accuracy],
-        callback_fns=[partial(FastAIV1PruningCallback, trial=trial, monitor="valid_loss")],
+        callback_fns=[partial(FastAIV1PruningCallback, trial=trial, monitor="accuracy")],
     )
     learn.fit(EPOCHS)
 
-    return learn.validate()[-1].item()
+    return learn.recorder.metrics[0][0].item()
```

Comment:
Please correct me if I'm wrong but my understanding is that metrics are reported by epoch. Are we correctly indexing the last epoch?

Reply:
Thank you for your comment! Your comment is absolutely right. The first index should be `-1` to return the validation accuracy at the last epoch.

Reply:
Thanks for the quick fix!